tests/provider/dwd/observation/test_api_data.py

```diff
@@ -558,25 +626,25 @@ def test_dwd_observation_data_result_tidy():
                         0,
                         # NM
                         pd.NA,
-                        8.0,
+                        100.0,
                         # VPM
                         pd.NA,
-                        6.4,
+                        640.0,
                         # PM
                         pd.NA,
-                        1008.60,
+                        100860.0,
                         # TMK
                         pd.NA,
-                        0.5,
+                        273.65,
```

Comment:
Is it true, when requesting `metric=True`, that this value, which is obviously in Celsius, will be returned in Kelvin? Isn't it just the wrong way round?

Reply:
You're actually right. My original intention was to name it SI which would be correct.

https://en.wikipedia.org/wiki/International_System_of_Units